client/src/components/Auth/Auth.css

```diff
@@ -0,0 +1,28 @@
+.App{
+    height: 100%;
+    width: 100%;
+}
+
+.App > h1{
+    font-family: "VAG World Bold";
+    margin: 0 25%;
+}
+
+.AuthForm{
+    display: grid;
+    grid-template-columns: repeat(1, minmax(210px, 1fr));
+    grid-gap: 32px;
+    width: 20%;
```

Comment:
![image](https://user-images.githubusercontent.com/95619588/228194823-047c5896-dc1d-4296-abaf-8531fb449430.png)
DO please max width :300 px or 200px, and set for inputs width: auto, it will look better than now(for login and signup pages)